### ISSUE TITLE
Remove dp_status from igoc.csv

### ISSUE THREAD

**Issue by rossbm** [TBS-EACPD/infobase#546]
The ternary variable `dp_status` in igoc.csv currently comes from Titan before being transformed by the pipeline. I think that the this variable might be used internally by Titan to decide what screens to present to departments, but I don't think that it is very useful for InfoBase purposes.

I would like to drop `dp_status` from igoc.csv. Could you look into removing references to it in InfoBase? If you are able to stop using it in InfoBase, let me know and I will drop it from the igoc.csv.

Thanks

**Comment by cookkate** [comment]
This is actually used for logic in InfoBase that decides whether to show planned spending or not, as well as in some results logic. If it's not causing trouble could we keep it? We'd prefer to have a firm source of truth on which orgs are submitting DPs via Titan rather than guessing from the data.

**Comment by rossbm** [comment]
I don't think that it is used for planned spending. Here's the current code for checking whether planned spending should be shown or not (Stephen is tweaking it slightly):

```
get has_planned_spending(){
    return !(
      _.includes([
        "CSEC", 
        "CSIS",
        "FCAC",
        "IJC",
        "GG",
      ],this.dept_code) ||
      _.includes([ 
        "Crown Corporations", 
        "Sociétés d'État", 
        "Parliamentary Entities",
        "Entités Parlementaires",
      ],this.inst_form)
    );
  }
```
There might be a few places in InfoBase where it is used, but there are probably better alternatives. One issue is if a department starts off a DP department, and then changes to a non-DP department, for example Leaders's Debate Commision. 

One thing we definitely do is switch it from a ternary variable to a a binary variable. I don't think that the distinction between first and second wave is needed anymore.
